schemas/sublime-completions.json

```diff
@@ -32,11 +56,44 @@
             "properties": {
               "trigger": {
                 "type": "string",
-                "markdownDescription": "Text that will be displayed in the completions list and will cause the contents to be inserted when chosen.\nYou can optionally follow it with a tab character (`\\t`) and a text to add a hint for the preceding trigger. The hint will be displayed alongside the trigger and does not affect the trigger itself."
+                "markdownDescription": "**Completion Trigger**\n\nThe text the user must enter to match the completion.\nIt can optionally be followed by a tab character (`\\t`) and an annotation.\n\n```js\n[\n    { \"trigger\": \"function\", ... },\n    { \"trigger\": \"variable\\tannotation\", ... },\n]\n```\n\n**Annotations:**\n\n- are displayed on the right-hand edge of the completions popup,\n  and may contain any information the author deems useful.\n- typically consist of a word or two.\n- can also be defined by dedicated `annotations` field _(requires ST4073)_."
```

Comment:
This string mentions that ST4073 is required for one thing but all these features are ST4+ really so IMO we don't need that note.

We should either just completely omit that detail or annotate all ST4-only features properly.